server/src/main/java/io/unitycatalog/server/service/AuthService.java

```diff
@@ -185,9 +185,12 @@ public HttpResponse logout(HttpRequest request) {
             authorizationCookie -> {
               Cookie expiredCookie = createCookie(AuthDecorator.UC_TOKEN_KEY, "", "/", "PT0S");
               ResponseHeaders headers =
-                  ResponseHeaders.of(
-                      HttpStatus.OK, HttpHeaderNames.SET_COOKIE, expiredCookie.toSetCookieHeader());
-              return HttpResponse.of(headers);
+                  ResponseHeaders.builder()
+                      .status(HttpStatus.OK)
+                      .add(HttpHeaderNames.SET_COOKIE, expiredCookie.toSetCookieHeader())
+                      .contentType(MediaType.JSON)
+                      .build();
+              return HttpResponse.of(headers, HttpData.ofUtf8("{}"));
```

Comment:
Can you just add a comment why we're sending an empty Json for now? like - Armeria needs a response payload to complete the request, so we are sending an empty JSON.

Reply:
addressed